provider/pkg/helm/tool.go

```diff
@@ -167,8 +166,6 @@ type TemplateCommand struct {
 	Validate    bool
 	IncludeCRDs bool
 	SkipTests   bool
-	KubeVersion string
-	ExtraAPIs   []string
```

Comment:
These two properties were unnecessary wrappers over properties in the embedded type (`action.Install`), and made it harder than necessary to setup the capabilities.